testnet/with_pristine_e2e_environment_test.go

```diff
@@ -120,6 +122,8 @@ func withPristineE2EEnvironment(t *testing.T, cb func(
 		Description: "stake",
 		Display:     "mstake",
 		Base:        "stake",
+		Name:        "mstake",
```

Comment:
If a test is reading in these values and we want to confirm they are being used rather than `Display`, we should make the value different.

Reply:
updated